libs/api/domains/driving-license/src/lib/drivingLicense.module.ts

```diff
@@ -1,13 +1,22 @@
 import { Module, DynamicModule } from '@nestjs/common'
 
-import { MainResolver, QualityPhotoResolver } from './graphql'
```

Comment:
Lint warning present in this file for `DynamicModule` defined but never used. Since we're editing it, might be opportune to fix the warning while we're at it :)

Reply:
Yes good spot!